packages/text/src/css/index.ts

```diff
@@ -107,31 +107,31 @@ export default {
     color: colorsTextIcon.lowOnDark
   },
   [`.psds-text__label--size-${vars.labelSizes.xSmall}`]: {
-    fontSize: '12px',
-    letterSpacing: '0.02em',
+    fontSize: type.fontSize100,
+    letterSpacing: type.letterSpacingWidest,
```

Comment:
Looks like this var doesn't exist on core.  Maybe it should? Maybe never? Not sure the scale you've decided on.

Getting:

```
$ tsc --project tsconfig.build.json --module commonjs --target es5
lerna ERR! yarn run build exited 1 in '@pluralsight/ps-design-system-text'
lerna ERR! yarn run build stdout:
$ run-s build:cjs build:esm build:css
$ tsc --project tsconfig.build.json --module commonjs --target es5 --outDir dist/cjs
src/css/index.ts(111,25): error TS2551: Property 'letterSpacingWidest' does not exist on type '{ fontFamily: string; fontFamilyCode: string; fontSizeGigantic: string; fontSizeJumbo: string; fontSizeXXLarge: string; fontSizeXLarge: string; fontSizeLarge: string; fontSizeMedium: string; ... 50 more ...; lineHeightHuge: string; }'. Did you mean 'letterSpacingWide'?
src/css/index.ts(116,25): error TS2551: Property 'letterSpacingWidest' does not exist on type '{ fontFamily: string; fontFamilyCode: string; fontSizeGigantic: string; fontSizeJumbo: string; fontSizeXXLarge: string; fontSizeXLarge: string; fontSizeLarge: string; fontSizeMedium: string; ... 50 more ...; lineHeightHuge: string; }'. Did you mean 'letterSpacingWide'?
src/css/index.ts(150,25): error TS2551: Property 'letterSpacingWidest' does not exist on type '{ fontFamily: string; fontFamilyCode: string; fontSizeGigantic: string; fontSizeJumbo: string; fontSizeXXLarge: string; fontSizeXLarge: string; fontSizeLarge: string; fontSizeMedium: string; ... 50 more ...; lineHeightHuge: string; }'. Did you mean 'letterSpacingWide'?
src/css/index.ts(190,25): error TS2551: Property 'letterSpacingWidest' does not exist on type '{ fontFamily: string; fontFamilyCode: string; fontSizeGigantic: string; fontSizeJumbo: string; fontSizeXXLarge: string; fontSizeXLarge: string; fontSizeLarge: string; fontSizeMedium: string; ... 50 more ...; lineHeightHuge: string; }'. Did you mean 'letterSpacingWide'?
```

Reply:
changed references to wide, wider to loose, looser (widest was removed).